k8s/examples/zookeeper/89-zookeeper-client.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: zookeeper-test-client
+  namespace: default
```

Comment:
Could we please remove the namespace: default.

Reply:
removed